scripts/cibuild.rb

```diff
@@ -105,6 +118,33 @@ def verify_file (f)
       return [f, error]
     end
 
+    if github_link?(link) then
+
+      if link.index('https://github.com/issues?q=') then
```

Comment:
Yeah, I'll relax this - I didn't get any false positives from this but who knows what the future holds?!?!?!